packages/e2e-tests/specs/performance/utils.js

```diff
@@ -110,6 +111,7 @@ export async function getLoadingDurations() {
 				).startTime - responseEnd,
 			// This is evaluated right after Puppeteer found the block selector.
 			firstBlock: performance.now() - responseEnd,
+			timeToFirstByte: responseStart - startTime,
```

Comment:
Nit: We probably don't need to subtract `startTime` here as it should [always be 0](https://developer.mozilla.org/en-US/docs/Web/API/PerformanceEntry/startTime).

Reply:
https://github.com/WordPress/gutenberg/pull/47541/commits/f0bc52f1334f450928c0e1ec936cf0d46b70feca